app/views/restrooms/_restroom.html.haml

```diff
@@ -24,7 +24,9 @@
     .itemDistance
       - if restroom.respond_to?(:distance)
         = restroom.distance.round(2)
-        miles
+        miles /
+        = (restroom.distance * 1.609344).round(2)
```

Comment:
Can we put this in a helper somewhere? And define the 1.609344 inside the helper as a constant. Rather than just having a hardcoded amount here?

http://mixandgo.com/blog/the-beginner-s-guide-to-rails-helpers has an example of what i'm talking about. It would be something like this:

file_name: `app/helpers/restrooms_helper.rb`

```ruby
module RestroomHelper
  KILO_DISTANCE_CONVERSION = 1.609344
  def miles_to_kilometers(miles)
    miles * KILO_DISTANCE_CONVERSION
  end
end
```

then you should be able to use it here just by calling
`miles_to_kilometers(restroom.distance).round(2)`